assets/l10n/app_en.arb

```diff
@@ -459,9 +467,13 @@
   "@loginAddAnAccountPageTitle": {
     "description": "Title for page to add a Zulip account."
   },
-  "loginServerUrlInputLabel": "Your Zulip server URL",
-  "@loginServerUrlInputLabel": {
-    "description": "Input label in login page for Zulip server URL entry."
+  "loginServerUrlLabel": "Your Zulip server URL",
+  "@loginServerUrlLabel": {
+    "description": "Label in login page for Zulip server URL entry."
+  },
+  "loginServerUrlHint": "your-org.zulipchat.com",
+  "@loginServerUrlHint": {
+    "description": "Hint text in login page for Zulip server URL entry."
```

Comment:
… Hmm. I started writing in my head a bit more for this description (the translation should keep the ".zulipchat.com" part), then realized: we actually can't have translators substituting their own translations of the phrase "your org" either.

That's because it's important that an example like this be [a name we've reserved](https://chat.zulip.org/#narrow/channel/243-mobile-team/topic/flutter.3A.20login.20URL/near/1570360) so that it won't be some actual Zulip organization.

So I'll drop this commit and instead write a comment saying why this isn't translated.

Reply:
Interesting. TIL!